### ISSUE TITLE
Problem: uncompressed AIPs need pointer files

### ISSUE THREAD

**Issue by jrwdunham** [artefactual/archivematica-storage-service#324]
Pointer files are needed to describe uncompressed AIPs. (They are already created for compressed AIPs.) Motivations:

1. Encrypted uncompressed AIPs (which are tarred) need pointer files so we can know which GPG key is needed to decrypt them.
2. Replica uncompressed AIPs need pointer files so we can know what original AIP is their source.
   - Relatedly, uncompressed AIPs cannot currently be replicated. AIP replication requires a pointer file. 
3. Improving the performance of fixity checking requires pointer files for uncompressed AIPs so that bag-based fixity checking of the entire AIP can be documented in the pointer files of all kinds of AIPs. See https://github.com/artefactual/archivematica/issues/918.

The scope of fixing this issue should include introduction of METS/PREMIS validation using the validation already implemented in mets-reader-writer.

TODO: provide an estimate of what resolving this issue would entail.

**Comment by joel-simpson** [comment]
Adding some context and additional detail following a brief discussion with @sromkey.... one of the reasons uncompressed AIPS did not have pointer files was due to a limitation in earlier versions of PREMIS, where you could only describe a bitstream, but not an intellectual entity. PREMIS 3 now allows for description of intellectual entity. 

I would add a further motivation for this fix... simple consistency. It would simply make more sense to have a pointer file for every AIP (one less 'exception' or special case for users to have to understand).  Alternatively, the UI should explain why there isn't a pointer file, or at least not provide a button to view the pointer file when one doesn't exist (currently if you press that button for an uncompressed AIP you get a 404 message from the storage service which is not very user-friendly).

**Comment by sallain** [comment]
@ross-spencer Assigning this to you for the future - it's for 1.12, but probably needs some analysis!

**Comment by jens-st** [comment]
I agree with @joel-simpson, more consistency for AIPs and less confusion for dashboard users.
Clicking on Pointer file --> "View" on a previous selected AIP still results in a 404 (not found) error for the user in Archivematica 1.13.0.

`{"success": false, "message": "Storage service returned 404; check logs?"}`
